modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,479 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-create-objects_{context}"]
+= Create objects
+
+[role="_abstract"]
+This section describes the `create` commands for clusters and resources.
+
+[id="rosa-create-admin_{context}"]
+== create admin
+
+Creates a cluster administrator with an automatically generated password that can log in to a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create admin --cluster=<cluster_name>
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to add to the identity provider (IDP).
```

Comment:
Required: 
You're likely to run into this comment a lot in the next topics, sorry. Probably best to make a one-time fix to all occurrences to follow our recommended style.

Reply:
Global fix implemented.